src/components/Home/index.tsx

```diff
@@ -21,11 +22,11 @@ const Home: React.FC = () => {
   return (
     <>
       <WhatsNewModal />
-      <HeroSection className={styles.heroSection}>
-        <LandingHero />
+      <Hero />
+      <HeroContainer className={styles.heroContainer}>
```

Comment:
The company logos section should have a background color of #EEF4F8 to provide more visual separation from the Get Started with DVC section above.